src/main/kotlin/cli/gitMinersCLI/ComplexityCodeChangesCLI.kt

```diff
@@ -78,16 +85,17 @@ class ComplexityCodeChangesCLI : GitMinerMultithreadedOneBranchCLI(
     }
 
     override fun run() {
-        val miner = ComplexityCodeChangesMiner(
-            repository,
-            branch,
-            numThreads = numThreads,
+        val dataProcessor = ComplexityCodeChangesDataProcessor(
             numOfMonthInPeriod = numOfMonth,
             numOfCommitsInPeriod = numOfCommits,
             changeType = changedTypeStringToEnum[changeType]!!,
             periodType = periodTypeStringToEnum[periodType]!!
         )
-        miner.run()
-        miner.saveToJson(resources)
+
+        val miner = ComplexityCodeChangesMiner(repository,branch,numThreads = numThreads)
```

Comment:
Please use "reformat code" before commit, or consider setting up a linter in CI.

Reply:
Done